documentation/src/main/asciidoc/topics/proc_enabling_spring_session.adoc

```diff
@@ -12,3 +12,9 @@ Complete this procedure to use Spring Session in your project.
 - `@EnableCaching`
 - `@EnableInfinispanRemoteHttpSession`
 - `@EnableInfinispanEmbeddedHttpSession`
+
+[NOTE]
+====
+{brandname} does not provide a default cache. To use Spring Session, you must
+add a {brandname} cache configuration.
+====
```

Comment:
Can we add a link to the docs that show how todo this?

Reply:
yeah, good call. thanks!